src/screens/AboutScreen.tsx

```diff
@@ -1,15 +1,279 @@
+
+const profileImage: ImageSourcePropType = require("../assets/img/DummySpeakerProfilePicture.jpeg");
+
+//Temporary function to make dummy data for the organizing team
+const makeDummyData = (n = 17) => {
+	let data = [];
+	for (let i = 1; i <= n; i++) {
+		data.push({
+			id: i.toString(),
+			name: `Member ${i}`,
+			title: `Title ${i}`,
+			image: profileImage,
+		});
+	}
+	return data;
 };
 
-export default AboutScreen;
\ No newline at end of file
+const MOCK_DATA_ORGANIZING_TEAM = makeDummyData();
+
+const AboutScreen = ({
+	navigation,
+}: NativeStackScreenProps<
+	RootStackParamList,
+	screen_names.ABOUT,
+	undefined
+>) => {
+	return (
+		<SafeAreaView style={[styles.container, styles.paddingVertical]}>
+			<StatusBar
+				backgroundColor={colors.DROIDCONKE_WHITE}
+				barStyle="dark-content"
+			/>
+			<View
+				style={[
+					styles.header,
+					styles.marginVerticalSeparator,
+					styles.paddingHorizontal,
+				]}
+			>
+				<DroidconKeIcon width={150} style={styles.droidconkeIcon} />
+				<View
+					style={[
+						layoutProperties.flexRow,
+						layoutProperties.justifyBetween,
+						layoutProperties.itemsCenter,
+					]}
+				>
+					<TouchableOpacity style={styles.buttonFeedback}>
+						<Image
+							resizeMode="contain"
+							source={require("../assets/icons/SmileyIcon.png")}
+							style={styles.buttonFeedbackContentMargin}
+						/>
+						<Text
+							style={[
+								styles.buttonFeedbackText,
+								styles.buttonFeedbackContentMargin,
+							]}
+						>
+							Feedback
+						</Text>
+						<Image
+							resizeMode="contain"
+							source={require("../assets/icons/SendIcon.png")}
+						/>
+					</TouchableOpacity>
+					<TouchableOpacity style={styles.iconWrapper}>
+						<Image
+							resizeMode="contain"
+							source={require("../assets/img/profilepicture.png")}
+						/>
+					</TouchableOpacity>
+				</View>
+			</View>
+			<ScrollView showsVerticalScrollIndicator={false}>
+				<Image
+					source={require("../assets/img/about.png")}
+					resizeMode="contain"
+					style={styles.heroImage}
+				/>
+				<View style={[styles.paddingHorizontal, styles.paddingVertical]}>
+					<Text style={[styles.headingText]}>About</Text>
+					<Text style={[styles.descriptionText]}>{introText}</Text>
+					<Text style={[styles.headingText]}>Organizing Team</Text>
+					<View style={[styles.teamContainer]}>
+						{MOCK_DATA_ORGANIZING_TEAM.map((member) => (
+							<TeamMemberCard
+								name={member.name}
+								title={member.title}
+								profileImage={member.image}
+								onPress={() =>
+									navigation.navigate(screen_names.BIO, {
+										bioData: {
+											...MOCK_BIO,
+											screenTitle: ScreenTitle.Team,
+											title: member.title,
+											name: member.name,
+											img: member.image,
+										},
+									})
+								}
+								key={member.id}
+							/>
+						))}
+					</View>
```

Comment:
The padding around the organizing team container is a little bit much. You can try decreasing it so that the organizing team container padding and `Organizing Team` title is uniform.
![photo_2022-10-10_10-58-41](https://user-images.githubusercontent.com/28436092/195012861-7d10c993-cb5a-4345-8121-d97ed1a8490c.jpg)

Reply:
> The padding around the organizing team container is a little bit much. I am noticing the [design](https://xd.adobe.com/view/dd5d0245-b92b-4678-9d4a-48b3a6f48191-880e/screen/c87e6ded-5f2e-4f62-81f1-3940e677d79f/specs/) also shows the same so this is **TOTALLY OPTIONAL**. You can try decreasing it so that the organizing team container padding and where the `Organizing Team` title starts is uniform. ![photo_2022-10-10_10-58-41](https://user-images.githubusercontent.com/28436092/195012861-7d10c993-cb5a-4345-8121-d97ed1a8490c.jpg)

Hi Brian, on this issue, I have tried various work-arounds and this seems a hard nut to crack. The spacing is auto-generated.

My alternatives were:
1. Use `justifyContent: "space-between"` on the `teamContainer`. This would solve the padding issue on the edges, but for rows with only 2 items, they are pushed to the edges leaving a huge space between them, which is inconsistent with the design.
2. Where a row has 2 items, create a third dummy object with its `display` property set to `none`. This would solve the issue completely, but then begs the question, is it good practice to introduce 'foreign/dummy' objects/data to make the layout behave in a certain way? I'd want to hear your thoughts on this. 
3. Given that each row has 3 items, give each item a width of 33.3% and let the flex box take care of distributing the remaining space. 


I chose option 3 given that it resembles the design and does not introduce any *hacks* around it. 

That said, am open to ideas/assistance/other alternatives.

Reply:
For now, this is not really a priority so whatever you have done is satisfactory. We can leave this conversation open so that anyone who wants to work on it in the future can pick it up

Reply:
Okay. That's in order.